### ISSUE TITLE
resert data or keep data

### ISSUE THREAD

**Issue by SimonLab** [postgres-plugin/people#3]
Add an option which tell the plugin if it needs to reset the tables or to just keep the current data.

**Comment by SimonLab** [comment]
When registering the plugin to an Hapi server, the user as the possibility to reset the database with new content define as arrays of object for each table:
```
{
reset: true // let the plugin know if the data should be reset
people: [{...},...],
organisations: [{...},...],
tags_organisations: [{...},...]
}
```
This allow a user to add easily new data without having to change manually a sql query.

**Comment by SimonLab** [comment]
resolved with #4
